And go the following error:
```
/usr/bin/ld: /data/balay/spack/opt/spack/linux-centos7-skylake_avx512/gcc-7.4.0/mfem-4.2.0-jegabnknxmqo44z26lz3v4q4rla4zztg/lib/libmfem.a(mem_manager.o): undefined reference to symbol 'cudaStreamSynchronize@@libcudart.so.10.1'
/nfs/apps/spacks/2021-02-09/opt/spack/linux-centos7-x86_64/gcc-7.3.0/cuda-10.1.243-kh3kzn4uiajmhy5oiofd6rmo7ube3q3p/lib64/libcudart.so.10.1: error adding symbols: DSO missing from command line
```
mfem requires a newer version of cuda? [but then some other xsdk packages won't build]

--- a/var/spack/repos/builtin/packages/xsdk-examples/package.py
+++ b/var/spack/repos/builtin/packages/xsdk-examples/package.py
@@ -21,7 +21,7 @@ class XsdkExamples(CMakePackage):
 
     variant('cuda', default=False, description='Compile CUDA examples')
 
-    depends_on('xsdk+cuda ^mfem+cuda', when='+cuda')
+    depends_on('xsdk+cuda', when='+cuda')
     depends_on('xsdk@0.6.0', when='@0.2.0')
     depends_on('xsdk@0.5.0', when='@0.1.0')
 

--- a/var/spack/repos/builtin/packages/xsdk-examples/package.py
+++ b/var/spack/repos/builtin/packages/xsdk-examples/package.py
@@ -50,9 +50,12 @@ class XsdkExamples(CMakePackage):
             '-DGINKGO_INCLUDE_DIR=%s' % spec['ginkgo'].prefix.include,
             '-DGINKGO_LIBRARY_DIR=%s' % spec['ginkgo'].prefix.include.lib,
             # allow use of default `find_package(Ginkgo)`
-            '-DCMAKE_PREFIX_PATH=%s/cmake' % spec['ginkgo'].prefix.include.lib,
-            '-DENABLE_CUDA=%s' % on_off('+cuda')
+            '-DCMAKE_PREFIX_PATH=%s/cmake' % spec['ginkgo'].prefix.include.lib
         ]
+        if '+cuda' in spec:
+            args.extend([
+                '-DENABLE_CUDA=ON'
+            ])
         if 'trilinos' in spec:  # if trilinos variant was activated for xsdk
             args.extend([
                 '-DTRILINOS_DIR_PATH=%s' % spec['trilinos'].prefix,

--- a/var/spack/repos/builtin/packages/xsdk-examples/package.py
+++ b/var/spack/repos/builtin/packages/xsdk-examples/package.py
@@ -50,7 +50,8 @@ class XsdkExamples(CMakePackage):
             '-DGINKGO_INCLUDE_DIR=%s' % spec['ginkgo'].prefix.include,
             '-DGINKGO_LIBRARY_DIR=%s' % spec['ginkgo'].prefix.include.lib,
             # allow use of default `find_package(Ginkgo)`
-            '-DCMAKE_PREFIX_PATH=%s/cmake' % spec['ginkgo'].prefix.include.lib
+            '-DCMAKE_PREFIX_PATH=%s/cmake' % spec['ginkgo'].prefix.include.lib,
+            '-DENABLE_CUDA=%s' % on_off('+cuda')
         ]
         if 'trilinos' in spec:  # if trilinos variant was activated for xsdk
             args.extend([

